Vikta-API-testing-sample/src/test/java/com/test/api/category/CategoryTest.java

```diff
@@ -62,6 +79,95 @@ public void checkCategoryInAllCategoriesList() {
                 hasItem(equalTo(expectedCategory)));
     }
 
+
+
+    /**
+     *  Test with entity search
+     */
+    // it's weird, when extract.as(CategoryDTO[].class) return empty array, at http://localhost:5054/swagger-ui.html it works
```

Comment:
check how to return ArrayList of given type from rest assured response and see if it will be empty too